learn/how-tos/integrating-server-side-pagination.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "Integrating Pagination for Imported APIs"
+id: "integrating-server-side-pagination"
+sidebar_label: "Pagination for Imported APIs"
+---
+---
+
+Pagination is a process that separates data into sets. Server-side pagination controls how these sets are retrieved from the database. It is useful when your application does not want to display All records at once. Instead, request data with a specific order or filter criteria the API sets out, such as Offset, Page and Cursor types. For example, Instagram feed with a Cursor pagination; it retrieves a set of five latest posts based on a timestamp, as you scroll.
+
+In WaveMaker, you can achieve Server-side pagination using the DSL (Domain Specific Language) method to the imported APIs, which works seamlessly with the data widgets. 
```

Comment:
Server side pagination is on the API. So this sentence should be matching the title 

In WaveMaker adding UI pagination for representing APIs that support server side pagination is possible using a little bit of JavaScript API exposed on the widgets such as List, Table.